codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -69,60 +69,113 @@
     private final Http2Settings initialSettings;
     private ChannelFutureListener closeListener;
     private BaseDecoder byteDecoder;
-    private long gracefulShutdownTimeoutMillis = DEFAULT_GRACEFUL_SHUTDOWN_TIMEOUT_MILLIS;
+    private long gracefulShutdownTimeoutMillis;
 
-    public Http2ConnectionHandler(boolean server) {
-        this(server, true);
+    public static final class Builder extends BuilderBase<Http2ConnectionHandler, Builder> {
```

Comment:
Should we split the builder out into its own class? Maybe we could merge the builder and the base class into a single class.  Something like `Http2ConnectionHandlerBuilder`, `Http2HandlerBuilder`, `Http2Builder` or `Http2CodecBuilder`?